source/db/en-projects-rejected.ls

```diff
@@ -290,14 +290,14 @@ projects-rejected =
     name: "Facebook"
     categories: [
       * name: "Web Services"
-        subcategories: ["Email Accounts", "Instant Messaging", "Social Networks", "Video & Voice"]
+        subcategories: ["Email Accounts", "Social Networks", "Video & Voice"]
       * name: "Servers"
-        subcategories: ["Email Server", "Instant Messaging", "Social Networks", "Video & Voice"]
```

Comment:
Yeah I know but you can still use the small chat boxes on facebook.com instead of having to use messenger. Would it too confusing/too much clutter to have both under instant messaging you guys think?